src/components/LocationDetails.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "../styles/LocationDetails.css";
+
+const LocationDetails = (props) => {
+  const { city, country, errorMessage } = props;
+  return errorMessage ? (
+    <h1>{errorMessage}</h1>
+  ) : (
+    <h1 className="location-details">{`${city}, ${country}`}</h1>
```

Comment:
To prevent the `,` from showing on the initial page load you could do something like 
```suggestion
    <h1 className="location-details">
      {`${city}
        <span className={city ? "displayComma" : "doNotDisplayComma}>,</span> 
      ${country}`}
    </h1>
```